platform/windows/display_server_windows.h

```diff
@@ -622,7 +622,11 @@ class DisplayServerWindows : public DisplayServer {
 	virtual void swap_buffers() override;
 
 	virtual void set_native_icon(const String &p_filename) override;
-	virtual void set_icon(const Ref<Image> &p_icon) override;
+	void window_set_icon(const WORD icon, WindowID p_window);
+
+	void window_set_icon(const Ref<Image> &p_icon, WindowID p_window);
+
+	HICON get_icon(WindowID p_window);
```

Comment:
What's the purpose of `HICON` getter/setter?

Reply:
Before I realized I this class had it's own method to create the main window icon I was going to go about it completely differently. These methods are redudant now.